app/controllers/sessions_controller.rb

```diff
@@ -1,19 +1,22 @@
 class SessionsController < ApplicationController
+  param == "1" ? remember(@user) : forget(@user)
```

Comment:
Rubocop báo lỗi này nên em fix gán kết quả câu điều kiện vào **param** rồi mới truyền biến **param** vào method **create** :
**app/controllers/sessions_controller.rb:4:3: C: Metrics/AbcSize: Assignment Branch Condition size for create is too high. [21.31/20]
  def create ...
  ^^^^^^^^^^**